src/client/pages/OfferDebugger/Debugger.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useCallback } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { ErrorBoundary, FallbackProps } from 'react-error-boundary'
+import { Button } from 'components/buttons'
+import { useCreateOnboardingQuoteCartMutation } from 'data/graphql'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { QuoteData } from './components/QuoteData'
+
+const Wrapper = styled.div`
+  max-width: 900px;
+  padding: 1rem;
+  margin: 0 auto;
+  color: ${colorsV3.gray100};
+`
+
+const Row = styled.div`
+  padding-bottom: 1rem;
+`
+
+const ErrorFallback: React.FC<FallbackProps> = ({
+  error,
+  resetErrorBoundary,
+}) => (
+  <div>
+    <h2>Something went wrong 😔</h2>
+    <div>
+      <p>Error message: {error.message}</p>
+      <pre style={{ whiteSpace: 'pre-wrap' }}>{error.stack}</pre>
+    </div>
+    <Button
+      background={colorsV3.red500}
+      foreground={colorsV3.gray100}
+      onClick={resetErrorBoundary}
+    >
+      👉 Try starting over by creating a new session!
+    </Button>
+  </div>
+)
+
+export const Debugger: React.FC = () => {
+  const locale = useCurrentLocale()
+  const [
+    createOnboardingQuoteCart,
+    { data },
+  ] = useCreateOnboardingQuoteCartMutation()
+  const quoteCartId = data?.onboardingQuoteCart_create
```

Comment:
I know this is not anything that can be done in this PR of course but it feels kind of weird that the `onboardingQuoteCart_create` just returns an ID as a string and not a data structure like {id: [the quote cart ID]}. Because when using this mutation it's not clear what the data you get back is, unless you check the schema 🤔 But it's nice to not always have to do that, and just get help by VS Code instead.

Reply:
Sure, please comment that to Tobias or directly in https://github.com/HedvigInsurance/rfc/pull/7!